go-controller/pkg/util/util.go

```diff
@@ -362,7 +362,8 @@ func GetActiveNetworkForNamespace(namespace string, nadLister nadlister.NetworkA
 	for _, nad := range namespaceNADs {
 		nadInfo, err := ParseNADInfo(nad)
 		if err != nil {
-			return activeNetwork, err
+			klog.Warningf("Skipping nad '%s/%s' as active network after failing parsing it with %v", nad.Namespace, nad.Name, err)
```

Comment:
the only other place we do this is in net_attach_def controller where the same thing is done of ignoring the error, and returning nil from that place

However, my concern with this design is:

1. We have a function called Parsing the NADs which returns an error that we can probably not do anything with except log it :P .
2. How do we ensure we don't trip in other places in the future when using ParseNADs - i.e the fact that this error must never be returned ever?.. :)
at the very least please change the `GetActiveNetworkForNamespace`'s func decsription and add a new line to its bottom talking about this logic of continuing if parseNAD returns error; what that means is if there was a real config problem like subnet wasn't set to a right value etc we'll just ignore processing that NAD even if its a "primary" NAD and pick a secondary NAD if any.